subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -2900,7 +2968,7 @@ void parse_channel_args_to_params(const struct shell *sh, int argc,
 	int opt_index = 0;
 	struct getopt_state *state;
 	static const struct option long_options[] = {
-		{"if-index", optional_argument, 0, 'i'},
+		{"iface", optional_argument, 0, 'i'},
```

Comment:
Shouldn't this be `required_argument`? At least the help text indicates that the interface value should be given.